test/extended/util/image/image.go

```diff
@@ -32,24 +32,30 @@ func init() {
 		"k8s.gcr.io/sig-storage/csi-attacher:v3.0.0":              -1,
 		"k8s.gcr.io/sig-storage/csi-node-driver-registrar:v1.2.0": -1,
 		"k8s.gcr.io/sig-storage/csi-node-driver-registrar:v1.3.0": -1,
+		"k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0": -1,
 		"k8s.gcr.io/sig-storage/csi-provisioner:v1.6.0":           -1,
 		"k8s.gcr.io/sig-storage/csi-provisioner:v2.0.0":           -1,
+		"k8s.gcr.io/sig-storage/csi-provisioner:v2.1.0":           -1,
 		"k8s.gcr.io/sig-storage/csi-resizer:v0.4.0":               -1,
 		"k8s.gcr.io/sig-storage/csi-resizer:v0.5.0":               -1,
+		"k8s.gcr.io/sig-storage/csi-resizer:v1.1.0":               -1,
 		"k8s.gcr.io/sig-storage/csi-snapshotter:v2.0.1":           -1,
 		"k8s.gcr.io/sig-storage/csi-snapshotter:v2.1.0":           -1,
 		"k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.2":           -1,
 		"k8s.gcr.io/sig-storage/hostpathplugin:v1.4.0":            -1,
 		"k8s.gcr.io/sig-storage/livenessprobe:v1.1.0":             -1,
 		"k8s.gcr.io/sig-storage/mock-driver:v4.0.2":               -1,
+		"k8s.gcr.io/sig-storage/mock-driver:v4.1.0":               -1,
 		"k8s.gcr.io/sig-storage/snapshot-controller:v2.1.1":       -1,
+		"k8s.gcr.io/sig-storage/snapshot-controller:v3.0.2":       -1,
 
 		// allowed upstream kube images - index and value must match upstream or
-		// tests will fail
-		"k8s.gcr.io/e2e-test-images/agnhost:2.21": 1,
-		"docker.io/library/nginx:1.14-alpine":     23,
-		"docker.io/library/nginx:1.15-alpine":     24,
-		"docker.io/library/redis:5.0.5-alpine":    31,
+		// tests will fail (vendor/k8s.io/kubernetes/test/utils/image/manifest.go)
+		"k8s.gcr.io/e2e-test-images/agnhost:2.30":       1,
```

Comment:
this is [slightly different ](https://github.com/kubernetes/kubernetes/blob/v1.21.0/test/utils/image/manifest.go#L216)upstream, that okay?

Reply:
https://github.com/kubernetes/kubernetes/blob/v1.21.0-rc.0/test/utils/image/manifest.go#L216 it'll require an update when we bump to 1.21.1, it changed after rc.0.

Reply:
OK. Point `6` from https://github.com/openshift/origin/blob/master/test/extended/util/image/README.md says:

`Don't introduce new versions of an existing image unless there is no choice - i.e. if you need redis and are not testing a specific version of redis, just use the existing image`

Reply:
Right, but this is coming from k8s, not from our tests :wink: